src/main/app/app.module.ts

```diff
@@ -1,12 +1,23 @@
 import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
-
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import {NavigationBarComponent} from "../navigation-bar/navigation-bar.component";
```

Comment:
please set you IDE setting to add space for each import so that we have a same space for all the imports of a component.